packages/teams-js/src/public/profile.ts

```diff
@@ -52,11 +52,10 @@ export namespace profile {
   /**
    * The type of the profile trigger.
    *  - MouseHover: The user hovered a target.
-   *  - MouseClick: The user clicked a target.
-   *  - KeyboardPress: The user initiated the show profile request with their keyboard.
+   *  - Press: The target was pressed with either a mouse click or keyboard key press.
```

Comment:
Should this also include touch events?

Reply:
Currently this API is only supported on Teams web and desktop, the plan is to keep this type minimal and only add new values when we see a need for them (such as when we start testing out integration in our first mobile hub)

Reply:
Right, but I assume this same value would also cover touch events in the future? Just trying to save you a future checkin where you have to update the documentation to say "and touch events", but touch events can't possibly trigger this yet then that's fine

Reply:
Ah sorry! I misunderstood, I thought you meant to add a new value for touch events. I think you're right that this value would cover touch events when we eventually add mobile support. I'll opt to update this doc header later, just in case I'm wrong and we actually have to do further changes to the API to support touch.